tests/on_wlm/test_simple_entity_launch.py

```diff
@@ -140,13 +140,18 @@ def test_summary(fileutils, test_dir, wlmutils):
     rows = [s.split() for s in summary_str.split("\n")]
     headers = ["Index"] + rows.pop(0)
 
+    # There is no guarantee that the order of
+    # the rows will be sleep, bad
     row = dict(zip(headers, rows[0]))
+    row_1 = dict(zip(headers, rows[1]))
+    if row["Name"] != sleep.name:
```

Comment:
minor nitpick - i worry about future testing adding `from time import sleep` and having a weird behavior due to overwriting a built-in. should we change name to avoid potential collision?

Reply:
Renamed!